RELEASE-NOTES.txt

```diff
@@ -1,7 +1,11 @@
 20.4
 -----
 * [*] Site Creation: Fixed a bug in the design picker where the horizontal position of designs could be reset. [#19020]
-
+* [*] Block Editor: Add React Native FastImage [https://github.com/WordPress/gutenberg/pull/42009]
```

Comment:
I think this should be an internal note, since it's a library addition and not a user-facing change. What do you think?

Reply:
Ah, yes, I agree, thank you for suggesting that! Updated in https://github.com/wordpress-mobile/WordPress-iOS/pull/19089/commits/a621fc8ee08b1bd9675888178e5d009edccdbf3e.